protocol/Subgraph/SubgraphData.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_position: 1
+title: Subgraph Data Introduction
+---
+
+# Sablier Subgraph Introduction
+
+Sablier has a GraphQL API Endpoint hosted by [The Graph](https://thegraph.com/docs/about/introduction#what-the-graph-is) called a subgraph for indexing and organizing data from the Sablier smart contracts.
+
+This subgraph is can be used to query Sablier data.
+
+Subgraph information is serviced by a decentralized group of server operators called Indexers.
+
+## Ethereum Mainnet
+
+[Creating an API Key Video Tutorial](https://www.youtube.com/watch?v=UrfIpm-Vlgs)
+
+- [Explorer Page](https://thegraph.com/explorer/subgraph?id=A5XzhYZ4muyRegVTYmwWbCbKWvD4LTWqac43CEGTEGdK&view=Playground)
+- Graphql Endpoint: https://gateway.thegraph.com/api/[api-key]/subgraphs/id/A5XzhYZ4muyRegVTYmwWbCbKWvD4LTWqac43CEGTEGdK

```

Comment:
Can we write this as "GraphQL"?

Reply:
Only reason why I would recommend against a hyperlink is the API key still needs to be input before it's a usable endpoint.

Thoughts?

Reply:
Oh, I see. In this case, I would recommend explaining this tidbit of information explicitly in a sentence. See how I refactored this in my latest commit.